### ISSUE TITLE
Move unimplemented stats to webrtc-provisional-stats

### ISSUE THREAD

**Issue by henbos** [w3c/webrtc-stats#621]
After several years of various metrics staying unimplemented, it's time to move them out of the main spec document in order to unblock webrtc-stats being able to reach Proposed Recommendation.

**Comment by Sean-Der** [comment]
Hey @henbos 

I don't know where things like this are discussed, but I would be happy to help implement these! I have always been looking for/interested in getting involved. 

Do you think these getting removed because they really aren't useful or just can't find an owner?

**Comment by alvestrand** [comment]
For instance, rtcpTransportStatsId is not getting implemented because nobody uses a separate transport for rtcp.
It's specified in the specs that it's possible to negotiate a WebRTC connection using a separate RTCP connection, but nobody does it. (I'm not sure if it works or not, in fact).

**Comment by henbos** [comment]
> Do you think these getting removed because they really aren't useful or just can't find an owner?

A little bit of both. The fact that nobody has implemented them is a sign that they're not that important (people seem to survive without them), but there may be exceptions to this.

Patches are welcome, but considering the large number of unimplemented metrics, I'd make sure there is a solid use case for the metric you want to implement.

If we move a metric over to [the provisional spec](https://w3c.github.io/webrtc-provisional-stats/) and it later gains implementation support, we can always move it back into this spec when it has matured.

**Comment by henbos** [comment]
I think we can close this issue now, but if there's anything else we missed we can file another issue.
